market/mk12/cidgravity.go

```diff
@@ -98,9 +100,10 @@ type cidGravityResponse struct {
 	MatchingRule            int    `json:"matchingRule"`
 }
 
-const cidGravityUrl = "https://service.cidgravity.com/api/proposal/check"
+const cidGravityDealCheckUrl = "https://service.cidgravity.com/api/proposal/check"
+const cidGravityMinerCheckUrl = "https://service.cidgravity.com/private/v1/miner-status-checker/check"
+const cidGravityMinerCheckLabel = "cidg-miner-status-check"
 const agentName = "curio"
-const formatVersion = "2.3.0"
 
 var commonHeaders = http.Header{
 	"X-Agent":         []string{"curio-market-storage-filter"},
```

Comment:
I forgot this in the first review, can you rename the common headers:

`X-Agent` to `X-CIDgravity-Agent`, and value must be `CIDgravity-storage-Connector`
`X-Agent-Version` to `X-CIDgravity-Version`